pinot-core/src/main/java/org/apache/pinot/core/segment/index/readers/BaseImmutableDictionary.java

```diff
@@ -81,6 +80,31 @@ public int indexOf(String stringValue) {
     return (index >= 0) ? index : NULL_VALUE_INDEX;
   }
 
+  @Override
+  public IntSet getDictIdsInRange(String lower, String upper, boolean includeLower, boolean includeUpper) {
+    throw new UnsupportedOperationException();
```

Comment:
why can this be supported?

Reply:
This method is for the unsorted dictionary. Added the note